EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc

```diff
@@ -68,12 +65,11 @@ class L1TValidationEventFilter : public edm::EDFilter {
 // constructors and destructor
 //
 L1TValidationEventFilter::L1TValidationEventFilter(const edm::ParameterSet& iConfig) :
+  src_(iConfig.getParameter<edm::InputTag>("src")),
   period_( iConfig.getUntrackedParameter<int>("period", 107) )
 {
   //now do what ever initialization is needed
 
-  fedData_ = consumes<FEDRawDataCollection>(iConfig.getParameter<edm::InputTag>("inputTag"));
```

Comment:
In principle, you are right, `getByToken` should be faster than `getByLabel`.  Can switch to it.